frontend/packages/core/src/stories/chip.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import type { Meta } from "@storybook/react";
+
+import type { ChipProps } from "../chip";
+import Chip from "../chip";
+
+export default {
+  title: "Core/Chip",
+  component: Chip,
```

Comment:
Storybook is going to see the variant as a string because of the typing system but if we add this it'll use a select:
```suggestion
  component: Chip,
  argTypes: {
    variant: {
      control: {
        type: 'select',
        options: [{options...}]
      }
    }
  }
```

If you want to reuse the options you can do something like (noted as suggestions as STORYBOOK TYPES):
```suggestion
  component: Chip,
  argTypes: {
    variant: {
      control: {
        type: 'select',
        options: CHIP_VARIANTS
      }
    }
  }
```

Reply:
it only seems to work if I do `as unknown as Meta` .  Not sure if thats the right way to go about things.  Also, seem to have a new issue with the < Chip {...props} part
![err](https://user-images.githubusercontent.com/66325812/115638910-28681200-a2c8-11eb-8299-defa43e527a1.gif)